internal/worker/cmd_meta.go

```diff
@@ -169,16 +169,16 @@ var CommandsMeta = map[string]CmdMeta{
 	// Multi-shard commands.
 	CmdRename: {
 		CmdType:            MultiShard,
-		preProcessingReq:   true,
+		preProcessing:      true,
 		preProcessResponse: preProcessRename,
 		decomposeCommand:   decomposeRename,
 		composeResponse:    composeRename,
 	},
 
 	CmdCopy: {
 		CmdType:            MultiShard,
```

Comment:
What do you think about converting this into a single shard command:
i.e. 
prepocessing = GetObject
cmd = SetObject

similar for `RENAME`.

Reply:
Considering the decompose for rename, it still have a multishard operations, del of k1 and set of k2.

Reply:
Let's discuss this over. I wanted to get your opinion on one of the approaches in mind.